python/grass/temporal/base.py

```diff
@@ -829,7 +829,8 @@ def print_info(self):
         """Print information about this class in human readable style"""
         #      0123456789012345678901234567890
         print(
-            " +-------------------- Basic information -------------------------------------+"
+            " +-------------------- Basic information ---------------------------------\
```

Comment:
This should not be split.

Reply:
OK. I got the rule.  I will make change "\" to style below.
```
s = ("this is a very "
      "long string"
     )
```

Reply:
In this particular case keep it on one line with noqa.